docker/kyuubi-service.yaml

```diff
@@ -0,0 +1,41 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Usage:
+# Start:
+#   kubectl apply -f https://raw.githubusercontent.com/apache/incubator-kyuubi/master/docker/kyuubi-service.yaml
+# Connect:
+#   ${SPARK_HOME}/bin/beeline -u 'jdbc:hive2://${hostname}:30009'
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: kyuubi-example-service
+spec:
+  ports:
+    # The default port limit is 30000-32767
+    # to change:
+    #   vim kube-apiserver.yaml (usually under path: /etc/kubernetes/manifests/)
+    #   add or change line 'service-node-port-range=1-32767' under kube-apiserver
+    - nodePort: 30009
+      # same of containerPort in pod yaml
+      port: 10009
+      protocol: TCP
+  type: NodePort
+  selector:
+    # some of pod label
```

Comment:
what's the meaning of `some of pod label`?

Reply:
Spelling mistakes, modified.